sql/hive/src/main/scala/org/apache/spark/sql/hive/client/ClientWrapper.scala

```diff
@@ -354,6 +354,25 @@ private[hive] class ClientWrapper(
     qlTable
   }
 
+  override def createView(view: HiveTable): Unit = withHiveState {
+    // TODO: this is duplicated with `toQlTable` except the table type stuff.
+    val tbl = new metadata.Table(view.database, view.name)
+    tbl.setTableType(HTableType.VIRTUAL_VIEW)
+    tbl.setSerializationLib(null)
+    tbl.clearSerDeInfo()
+    tbl.setViewOriginalText(view.viewText.get)
+    tbl.setViewExpandedText(view.viewText.get)
```

Comment:
Let's add a note at here to explain that we will save the same sql to original text and expanded text, which is not the case for Hive.